buildlib/azure-pipelines-pr.yml

```diff
@@ -48,6 +48,32 @@ stages:
               fi
             condition: eq(variables['Build.Reason'], 'PullRequest')
 
+      # Check that the code is formatted according to the code style guidelines
+      - job: format
+        displayName: format code
+        container: fedora
+        steps:
+          - checkout: self
+            clean: true
+
+          - bash: |
+              source ./buildlib/az-helpers.sh
+              echo "Checking code format on diff remotes/origin/${SYSTEM_PULLREQUEST_TARGETBRANCH}..${BUILD_SOURCEVERSION}"
+              git-clang-format --diff remotes/origin/${SYSTEM_PULLREQUEST_TARGETBRANCH} ${BUILD_SOURCEVERSION} > format.patch
+              echo "Generated patch file:"
+              cat format.patch
+              if [ "`cat format.patch`" = "no modified files to format" ]; then
+                  exit
+              fi
+              git apply format.patch
+              if ! git diff --quiet --exit-code
+              then
+                  url="https://github.com/openucx/ucx/wiki/Code-style-checking"
+                  azure_log_warning "Code is not formatted according to the code style, see $url for more info."
```

Comment:
this function is not defined in this PR, need to take from #5974
otherwise ok

Reply:
Why not? I see it: https://github.com/gleon99/ucx/blob/clang-format/buildlib/az-helpers.sh#L50

Reply:
right
so pls squash

Reply:
Squashed